user_tools/build.sh

```diff
@@ -174,7 +174,7 @@ build_jar_from_source() {
     bail "Failed to build the tools jar from source"
   fi
   # rename jar file stripping snapshot
-  TOOLS_JAR_FILE=( "$( find "${jar_dir}" -type f \( -iname "rapids-4-spark-tools_*.jar" ! -iname "*sources.jar" ! -iname "*tests.jar" ! -iname "original-rapids-4*.jar" ! -iname "*javadoc.jar" \) )" )
+  TOOLS_JAR_FILE=$(find "${jar_dir}" -type f \( -iname "rapids-4-spark-tools_*.jar" ! -iname "*sources.jar" ! -iname "*tests.jar" ! -iname "original-rapids-4*.jar" ! -iname "*javadoc.jar" \) | head -n 1)
```

Comment:
to fix warning: Expanding an array without an index only gives the first element. [SC2128]
TOOLS_JAR_FILE should contain only one file, get the first one of result of find command.

Reply:
Yes, there should be 1 result from the array.
The change LGTM!
Thanks @YanxuanLiu